lms/static/scripts/frontend_apps/components/BasicLtiLaunchApp.js

```diff
@@ -111,17 +110,20 @@ export default function BasicLtiLaunchApp({ clientRpc }) {
    * Fetch the groups from the sync endpoint if `sync` object exists.
    */
   const fetchGroups = useCallback(async () => {
-    if (apiSync) {
+    if (!apiSync) {
+      return true;
+    } else {
```

Comment:
You can avoid the `else` here since the `if` branch returns. This avoids an extra level of indentation. ie.

```js
if (!apiSync) {
  return true;
}

try {
  ...
}
```